src/m365/aad/commands/o365group/o365group-user-set.ts

```diff
@@ -64,7 +64,7 @@ class AadO365GroupUserSetCommand extends GraphItemsListCommand<GroupUser> {
           logger.logToStderr('');
         }
 
-        if (this.items.filter(i => i.userPrincipalName.toLocaleLowerCase() === args.options.userName.toLocaleLowerCase()).length <= 0) {
+        if (this.items.filter(i => args.options.userName.toUpperCase() === i.userPrincipalName!.toUpperCase()).length <= 0) {
```

Comment:
In the future, if there are specific issues that require code change, let's add some information to clarify it for future reference. Our CI/CD includes Linux tests that pass just fine, so right now it's unclear what's breaking exactly and why this change is necessary.